tribe-ext-selectively-show-html/src/functions/general.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
I'm a little confused about this existence of this file — `/src/functions/general.php` — and the above file, `/src/Tribe/functions/general.php`

The latter seems to basically just be the *exact duplicate* of the former, with the addition of `tribe_call_func()` and `tribe_is_ajax()`

Is there some reason these two files to both exist? If not, one should be deleted, it seems...